speid/models/transaction.py

```diff
@@ -118,19 +118,22 @@ def get_order(self):
             if snake_to_camel(k) in ORDEN_FIELDNAMES and (
                 trx_dict[k] is not None)
         }
+        order_dict['institucionOperante'] = self.institucion_ordenante
+        order_dict['institucionContraparte'] = self.institucion_beneficiaria
+        order_dict['nombreBeneficiario'] = (self.nombre_beneficiario[:38].
+                                            strip())
+        order_dict['nombreOrdenante'] = self.nombre_ordenante[:38].strip()
+        order_dict['monto'] = self.monto / 100.0
+
         order = Orden(**order_dict)
-        order.fechaOperacion = None
-        order.institucionOperante = self.institucion_ordenante
-        order.institucionContraparte = self.institucion_beneficiaria
-        order.nombreBeneficiario = self.nombre_beneficiario[:38].strip()
-        order.nombreOrdenante = self.nombre_ordenante[:38].strip()
+
         self.clave_rastreo = self.clave_rastreo or order.claveRastreo
         self.tipo_cuenta_beneficiario = self.tipo_cuenta_beneficiario or (
             order.tipoCuentaBeneficiario)
         self.rfc_curp_beneficiario = self.rfc_curp_beneficiario or (
             order.rfcCurpBeneficiario)
         self.referencia_numerica = self.referencia_numerica or (
             order.referenciaNumerica)
-        self.empresa = self.empresa or order.empresa
+        self.empresa = self.empresa or 'NA'
```

Comment:
I don't think its important, I can use the default one that we have instead of 'NA'. This field is actually not being used and we only have one `empresa`